peregrine/analysis/tracking_loop.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env python
```

Comment:
Could we also add an iPython notebook to demonstrate how to use the tools in this file and `peregrine/analysis/print_res.py`?